fern/docs/pages/snippet-templates.mdx

```diff
@@ -0,0 +1,15 @@
+import Template from "../snippets/code-template.mdx";
+
+Below are 2 code snippets that are templated using the `code-template` mdx-based component.
+
+<Template title="Example 1" components={props.components}>
```

Comment:
at the moment there's a bug where `components={props.components}` need to be passed in explicitly. We'll need to fix this on our side so it's handled automatically.